docs/training-run/index.txt

```diff
@@ -4,106 +4,255 @@
-configured environment and the submitted training code. We supply a set of default container images
-that are appropriate for many deep learning tasks, but users can also supply a :ref:`custom image
-<custom-docker-images>` if desired. If the specified container images are not existent locally, the
-trial container will fetch the images from the registry.
+When a trial is ready to run, the master communicates with the agent, or :ref:`distributed training
+<multi-gpu-training>` agents, which create(s) containers that have the configured environment and
+training code. A set of default container images applicable to many deep learning tasks is provided,
+but you can also specify a :ref:`custom image <custom-docker-images>`. If the specified container
+images do not exist locally, the trial container fetches the images from the registry.
 
-After starting the containers, each trial runs the ``startup-hook.sh`` that exists in the context
+After starting the containers, each trial runs the ``startup-hook.sh`` script in the context
 directory.
 
-However, this whole process might take very long before each trial starts training.
+The pre-training activity can take a long time before each trial begins training.
 
 ****************
  Start Training
 ****************
 
-To start training, we load and run the entrypoint, which is the user-defined Python class and
-specified in the experiment configuration.
-
-The user-provided class must be a subclass of a trial class included in Determined. Each trial class
-is designed to support one deep learning application framework. While training or validating models,
-the trial may need to load data from an external source therefore the training code needs to define
-data loaders.
+When training starts, the system launches the object or script specified by the entry point defined
+in the configuration file. Agents automatically communicate with the master to get what is needed to
+run and report results. Tasks might include:
 
-Our library automatically communicates with the master to get what needs to run and report back the
-results. It might:
-
-#. train the model for a few batches on the training dataset;
-#. checkpoint the states of the model and other objects;
-#. validate the model on the validation dataset.
+-  Train the model for a few batches on the training dataset.
+-  Checkpoint the model and other object states.
+-  Validate the model on the validation dataset.
 
 ********************
  Pause and Activate
 ********************
 
-An important feature of Determined is the ability to have trials stop running and then start again
-later without losing any training progress. The scheduler might choose to stop running a trial to
-allow a trial from another experiment to run, but a user can also manually pause an experiment at
-any time, which causes all of its trials to stop.
+Trials can be stopped and restarted without losing training progress. The scheduler can stop running
```

Comment:
let's say "trials can be paused and reactivated" since we use pause/stop buttons in the webui, where pause is the button that lets you continue, and the word "restart" could imply "from the beginning".  Also "activate" is the word we use in the cli and the python sdk for making a paused train start training again.